lib/build/bundled-source.js

```diff
@@ -14,11 +14,14 @@ exports.BundledSource = class {
     this.includedIn = null;
     this.includedBy = null;
     this._contents = null;
+    this._transformedSourceMap = null;
     this.requiresTransform = true;
   }
 
   get sourceMap() {
-    return this.file.sourceMap;
+    return this._transformedSourceMap === null
+      ? this.file.sourceMap
+      : this._transformedSourceMap;
```

Comment:
I think you can just do `return this._transformedSourceMap || this.file.sourceMap;`.

Reply:
I adopted the suggested change and I also added check when caching - if property `_transformedSourceMap` is `null`, cache value is set to `undefined`, so it's skipped in `JSON.stringify`. No need to cache `null` values.